src/upgrade.cpp

```diff
@@ -175,6 +175,16 @@ ConfigObject<ConfigValue>* Upgrade::versionUpgrade(const QString& settingsPath)
             delete oldFile;
         }
         else {
+#elif __WINDOWS__
+        qDebug() << "Config version is empty, trying to read pre-1.12.0 config";
```

Comment:
I'm not sure this will work correctly, because we did in fact version the config file since like 1.7 IIRC, so this code won't run on anyone's Mixxx config from a version between that and 1.12. At least not on English Windows XP.